packages/eslint-plugin/tests/rules/no-unnecessary-qualifier.test.ts

```diff
@@ -1,21 +1,19 @@
-import path from 'path';
-import rule from '../../src/rules/no-unnecessary-qualifier';
-import { RuleTester } from '../RuleTester';
 import { AST_NODE_TYPES } from '@typescript-eslint/experimental-utils';
+import rule from '../../src/rules/no-unnecessary-qualifier';
+import { RuleTester, getFixturesRootDir } from '../RuleTester';
 
-const messageId = 'unnecessaryQualifier';
-const rootPath = path.join(process.cwd(), 'tests/fixtures/');
+const rootPath = getFixturesRootDir();
 
 const ruleTester = new RuleTester({
   parser: '@typescript-eslint/parser',
   parserOptions: {
     tsconfigRootDir: rootPath,
     project: './tsconfig.json',
-    sourceType: 'module',
-    ecmaVersion: 6,
```

Comment:
eslint has internal validation for some ast and this is requred as we are using `import` within tests

es2015 = es6

> ImportDeclaration should appear when the mode is ES6 and in the module context.